mythril/laser/ethereum/instructions.py

```diff
@@ -68,10 +68,17 @@ def jumpdest_(self, global_state):
 
     @instruction
     def push_(self, global_state):
```

Comment:
I think this implementation is getting a bit too complicated. 
I think this can be fixed in a more simple way using Concat

Reply:
Concat is same as doing a "0" * len right?first i will try to simplify in a typical way and then lets see